swagger.yaml

```diff
@@ -158,6 +158,38 @@ paths:
       responses:
         204:
           description: OK
+  /asset:
+    get:
+      tags:
+        - Asset
+      description: Get an asset by id.
+      responses:
+        200:
+          description: OK
+        404:
+          description: Not Found
+    put:
+      tags:
+        - Asset
+      description: Upsert an asset.
+      responses:
+        204:
+          description: OK
+  /asset/url/{url}:
```

Comment:
I would prefer to see this in a more url safe manner as:

```
/asset?url={{ urlencoded_url }}
```

Because we can urlencode a query param and not a route param.